csharp/Sdk.Examples/Horizon/Tutorials/Instrument/Instrument.cs

```diff
@@ -0,0 +1,57 @@
+using System.Threading.Tasks;
+using Finbourne.Horizon.Sdk.Model;
+using NUnit.Framework;
+using Sdk.Examples.Horizon.Utilities;
+
+namespace Sdk.Examples.Horizon.Tutorials.Instrument
+{
+    [TestFixture]
+    public class Instrument : TutorialBase
+    {
+        [Test]
+        public async Task Search_OpenFigi_Query()
+        {
+            // Query OpenFigi
+            var openFigiQueryResult = await InstrumentApi.SearchOpenFigiAsync("VOD", false);
+            Assert.That(openFigiQueryResult.Results, Is.Not.Empty);
+        }
+
+        [Test]
+        public async Task Search_OpenFigi_Market_Sector()
+        {
+            // Check Equity is a valid Market Sector 
+            var allowedMarketSectors =
+                await InstrumentApi.GetOpenFigiParameterOptionAsync(OpenFigiParameterOptionName.MarketSector);
```

Comment:
minor formatting but can. you put this on one line to make it more readable (and the others)